From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: reexport tcp_send_ack
Patch-mainline: never, kabi
References: kabi

In 4.4.145, commit 0b1d40e9e7738e3396ce414b1c62b911c285dfa3 (tcp: do not
cancel delay-AcK on DCTCP special ACK), upstream commit
27cde44a259c380a3c09066fc4b42de7dde9b1ad removed export of tcp_send_ack.

Reexport the function in case somebody else uses that.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 net/ipv4/tcp_output.c |    1 +
 1 file changed, 1 insertion(+)

--- a/net/ipv4/tcp_output.c
+++ b/net/ipv4/tcp_output.c
@@ -3411,6 +3411,7 @@ void tcp_send_ack(struct sock *sk)
 {
 	__tcp_send_ack(sk, tcp_sk(sk)->rcv_nxt);
 }
+EXPORT_SYMBOL_GPL(tcp_send_ack);
 
 /* This routine sends a packet with an out of date sequence
  * number. It assumes the other end will try to ack it.
